src/middlewares/__tests__/createMetadataMiddleware.ts

```diff
@@ -13,28 +13,45 @@ declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace NodeJS {
     interface Global {
-      navigator: {
-        userAgent: string;
+      navigator?: {
+        userAgent?: string;
+        // Deprecated property, but still used in React Native
+        // https://github.com/facebook/react-native/blob/8bd3edec88148d0ab1f225d2119435681fbbba33/Libraries/Core/setUpNavigator.js
+        product?: string;
       };
       window: Window;
     }
   }
 }
 
 const { window } = global;
+
 Object.defineProperty(
   window.navigator,
   'userAgent',
   ((value) => ({
     get() {
       return value;
     },
-    set(v: string) {
-      value = v;
+    set(newValue) {
```

Comment:
Removed the typing as it doesn't have any effect. The above `namespace` does the type inference.

Reply:
Well, our linter doesn't like it apparently 😅

Reply:
Ignored the error, as it's not needed.